thunder/core/transforms.py

```diff
@@ -1854,14 +1854,16 @@ def _vmap_call_metafunc(detached: bool, args, in_dims, out_dims, axis_size, func
             return tree_unflatten(outs, spec)
         if isinstance(result, (Number, NumberProxy)) and axis_size is not None:
             # TODO: fetch the default device from the context
-            result = full(shape=(), fill_value=result, device=common_device)
+            result = ltorch.full(shape=(), fill_value=result, device=common_device)
```

Comment:
Hi @IvanYashchuk , I'd like to confirm here: I change it to `ltorch.full` infers dtype considering the Torch's default dtype, `clang.full` infers dtype directly from `fill_value` when `dtype=None`, so here in the `vmap` we should use ltorch right?